docs/src/network.md

```diff
@@ -619,7 +619,7 @@ page.route("**/title.html", route -> {
 ```
 
 ```python async
-async def handle_route(route: Route) -> None:
+async def handle_route(route: Route, _: Request) -> None:
```

Comment:
Is it actually required in python?

Reply:
Actually, no. Just dropped this "fix" from the CL.